notebooks/post_DL3_analysis.ipynb

```diff
@@ -1,36 +1,58 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        This method is further explained in doi:10.1088/0004-637X/707/2/1310
</pre><p>Add comment: In Gammapy 1.2 it is possible to calculate the pivot energy for any spectral model not just for the power law.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>